src/platforms/pc-stlinkv2/README.md

```diff
@@ -2,17 +2,21 @@ Stlink V2/3 with original STM firmware as Blackmagic Debug Probes
 
 Recent STM Stlink firmware revision (V3 and V2 >= J32) expose nearly all
 functionality that BMP needs. This branch implements blackmagic debug probe
-for the STM Stlink as a proof of concept.
+for the STM Stlink.
```

Comment:
The vendor writes the product name as "ST-LINK". Not sure if you want to follow that, just pointing it out.